inc/editor/namespace.php

```diff
@@ -131,35 +131,253 @@ function add_theme_supports() {
-		[ 'name' => __( 'Large', 'shiro-admin' ),   'shortName' => __( 'L', 'shiro-admin' ),  'size' => 24, 'slug' => 'large'   ],
-		[ 'name' => __( 'X-Large', 'shiro-admin' ), 'shortName' => __( 'XL', 'shiro-admin' ), 'size' => 32, 'slug' => 'xlarge'  ],
-		[ 'name' => __( 'Jumbo', 'shiro-admin' ),   'shortName' => __( 'J', 'shiro-admin' ),  'size' => 40, 'slug' => 'jumbo'   ],
+		[
+			'name' => __( 'Small', 'shiro-admin' ),
+			'shortName' => __( 'S', 'shiro-admin' ),
+			'size' => 14,
+			'slug' => 'small',
+		],
+		[
+			'name' => __( 'Medium', 'shiro-admin' ),
+			'shortName' => __( 'M', 'shiro-admin' ),
+			'size' => 20,
+			'slug' => 'medium',
+		],
+		[
+			'name' => __( 'Large', 'shiro-admin' ),
+			'shortName' => __( 'L', 'shiro-admin' ),
+			'size' => 24,
+			'slug' => 'large',
+		],
+		[
+			'name' => __( 'X-Large', 'shiro-admin' ),
+			'shortName' => __( 'XL', 'shiro-admin' ),
+			'size' => 32,
+			'slug' => 'xlarge',
+		],
+		[
+			'name' => __( 'Jumbo', 'shiro-admin' ),
+			'shortName' => __( 'J', 'shiro-admin' ),
+			'size' => 40,
+			'slug' => 'jumbo',
+		],
 	] );
 
 	// Remove the ability to set custom font sizes in the editor.
 	add_theme_support( 'disable-custom-font-sizes' );
 
 	// Define colors selectable in the editor.
 	add_theme_support( 'editor-color-palette', [
-		[ 'name' => __( 'Base 0', 'shiro-admin' ),    'slug' => 'base0',    'color' => '#000000' ],
-		[ 'name' => __( 'Base 10', 'shiro-admin' ),   'slug' => 'base10',   'color' => '#202122' ],
-		[ 'name' => __( 'Base 20', 'shiro-admin' ),   'slug' => 'base20',   'color' => '#54595d' ],
-		[ 'name' => __( 'Base 30', 'shiro-admin' ),   'slug' => 'base30',   'color' => '#72777d' ],
-		[ 'name' => __( 'Base 50', 'shiro-admin' ),   'slug' => 'base50',   'color' => '#a2a9b1' ],
-		[ 'name' => __( 'Base 70', 'shiro-admin' ),   'slug' => 'base70',   'color' => '#c8ccd1' ],
-		[ 'name' => __( 'Base 80', 'shiro-admin' ),   'slug' => 'base80',   'color' => '#eaecf0' ],
-		[ 'name' => __( 'Base 90', 'shiro-admin' ),   'slug' => 'base90',   'color' => '#f8f9fa' ],
-		[ 'name' => __( 'Base 100', 'shiro-admin' ),  'slug' => 'base100',  'color' => '#ffffff' ],
-		[ 'name' => __( 'Blue 50', 'shiro-admin' ),   'slug' => 'blue50',   'color' => '#3a25ff' ],
-		[ 'name' => __( 'Blue 90', 'shiro-admin' ),   'slug' => 'blue90',   'color' => '#eeeaff' ],
-		[ 'name' => __( 'Red 50', 'shiro-admin' ),    'slug' => 'red50',    'color' => '#d40356' ],
-		[ 'name' => __( 'Red 90', 'shiro-admin' ),    'slug' => 'red90',    'color' => '#fbe9f1' ],
-		[ 'name' => __( 'Yellow 50', 'shiro-admin' ), 'slug' => 'yellow50', 'color' => '#fffd33' ],
-		[ 'name' => __( 'Yellow 90', 'shiro-admin' ), 'slug' => 'yellow90', 'color' => '#fef6e7' ],
-		[ 'name' => __( 'Light Blue', 'shiro-admin' ), 'slug' => 'light-blue', 'color' => '#effafd' ],
-		[ 'name' => __( 'Wiki Blue', 'shiro-admin' ), 'slug' => 'wiki-blue', 'color' => '#3366CC' ],
+		[
+			'name' => __( 'Base 10', 'shiro-admin' ),
```

Comment:
Nice PHPCS cleanup, here, thanks for that.

It's always up to your discretion whether it would be easier to use `/* phpcs:disable The.Rule.Name */` comments for these sections, in case it's ever an annoying distraction in the future

Reply:
I've realized there's a merge conflict here with `main`, so when we address it @pamprn09 , which do you prefer -- spaced out like you have it now, on separate lines, or more condensed (with PHPCS ignore comments) like this?

```php
	/* phpcs:disable WordPress.Arrays.CommaAfterArrayItem */
	/* phpcs:disable WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound */
	add_theme_support( 'editor-color-palette', [
		[ 'name' => __( 'Base 10', 'shiro-admin' ),          'slug' => 'base10',           'color' => '#202122' ],
		[ 'name' => __( 'Base 20', 'shiro-admin' ),          'slug' => 'base20',           'color' => '#54595d' ],
		[ 'name' => __( 'Base 30', 'shiro-admin' ),          'slug' => 'base30',           'color' => '#72777d' ],
		[ 'name' => __( 'Base 50', 'shiro-admin' ),          'slug' => 'base50',           'color' => '#a2a9b1' ],
		// etcetera
	] );
	/* phpcs:enable */
```

Reply:
I prefer when things are easier to read, separated lines makes more sense to me. WDYT @kadamwhite ?